commands/subcount.js

```diff
@@ -0,0 +1,36 @@
+const { got } = require('./../got');
+
+module.exports = {
+	name: 'subcount',
+	description: 'Gets the subscriber count for a specified channel.',
+	ping: true,
+	permission: 100,
+	category: 'Info command',
+	noBanphrase: true,
+	execute: async (channel, user, input, perm) => {
+		try {
+			if (module.exports.permission > perm) {
+				return;
+			}
+
+			let channelName = input[2];
+			if (channelName.startsWith('@')) {
+				channelName = channelName.substring(1);
+			}
+
+			const url = `https://decapi.me/twitch/subcount/${channelName}`;
```

Comment:
This wont work, since the streamers would need to authenticate with the site first.
Subcount isn't public data